packages/make-angular-cli-faster/src/utilities/migration.ts

```diff
@@ -0,0 +1,182 @@
+import { getPackageManagerCommand, readJsonFile } from '@nrwl/devkit';
+import { output } from '@nrwl/workspace/src/utilities/output';
+import { execSync } from 'child_process';
+import { prompt } from 'enquirer';
+import { appRootPath } from 'nx/src/utils/app-root';
+import { lt, lte, major, satisfies } from 'semver';
+import { resolvePackageVersion } from './package-manager';
+import { MigrationDefinition } from './types';
+
+// up to this version the generator was in the @nrwl/workspace package
+const latestWorkspaceVersionWithMigration = '13.9.3';
```

Comment:
I think this would be better as `firstWorkspaceVersionWithoutMigration` in case we cut more patches of `13.9.x`

Reply:
I considered that, but I do need the latest version where the migration is still in the workspace package. I can change it to `~13.9.3` and then resolve the version.